packages/design-tokens/src/css/typography.css

```diff
@@ -128,15 +119,6 @@
   --typography-l-heading-sm-line-height: var(--line-height-4);
   --typography-l-heading-sm-font-weight: var(--font-weight-bold);
   --typography-l-heading-sm-letter-spacing: var(--letter-spacing-0);
-
-  /**
-   * @deprecated [#1] since version 1.9 [#2].
-   * [#3] Will be deleted in version 2.0.
-   */
-  --typography-l-heading-sm-regular-font-size: var(--font-size-5);
-  --typography-l-heading-sm-regular-line-height: var(--line-height-4);
-  --typography-l-heading-sm-regular-font-weight: var(--font-weight-regular);
-  --typography-l-heading-sm-regular-letter-spacing: var(--letter-spacing-0);
```

Comment:
Removing depreacted CSS variables from CSS stylesheet. Ensured they are not used in the codebase

<img width="504" alt="Screenshot 2025-05-25 at 2 20 39 PM" src="https://github.com/user-attachments/assets/ba573812-3966-4069-986d-4f4e658c8013" />